data/man/sddm.conf.rst.in

```diff
@@ -228,6 +228,12 @@ The `UserAuthFile=` option was removed, the file is always created as
 	won't be updated.
 	Default value is true.
 
+`ShowSavedUsers=`
```

Comment:
> Conceptually it might be an addition to this option by remembering more than one user?

This variant is possible. But, nevertheless, conceptually these options are slightly different. `RememberLastUser` remembers exactly the last user, while `ShowSavedUsers` saves users and does not care about their order in the list, which of them logged in earlier and which later.

In other words, if we implement `RememberLastUser` in a list of users that saves `ShowSavedUsers`, then we will either have to sort the list of saved users each time so that the last user is at the begin/end, or store the position of the last user in the list in a separate parameter.

In my opinion, all this complicates the logic.

Maybe just rename this 'ShowSavedUsers` to `RememberUsers`?